### PR TITLE
Fix testcontainers cql for hadoop

### LINKED ISSUE #1734: Branch 0.4 and master are failing on travis ci
Hadoop tests are failing.

### PR DESCRIPTION
**Pull request by farodin91**
Fixes #1734

Signed-off-by: Jan Jansen <jan.jansen@gdata.de>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there an issue associated with this PR? Is it referenced in the commit message?
- [x] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?
- [ ] If this PR is a documentation-only change, have you added a `[skip ci]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: janusgraph-cql/src/test/java/org/janusgraph/JanusGraphCassandraContainer.java [hunk 1]
```diff
@@ -95,10 +95,14 @@ private String getConfigPrefix() {
...
     }
+    public JanusGraphCassandraContainer(){
+        this(false);
+    }
 
-    public JanusGraphCassandraContainer() {
+    public JanusGraphCassandraContainer(boolean fixedExposedPortOfCQL) {
         super(getCassandraImage() + ":" + getVersion());
         withEnv("MAX_HEAP_SIZE", "2G");
+        addFixedExposedPort(CQL_PORT, CQL_PORT);
```
Comment (porunov) [inline janusgraph-cql/src/test/java/org/janusgraph/JanusGraphCassandraContainer.java]: Should if be covered by `if`?
```
if(fixedExposedPortOfCQL){
  addFixedExposedPort(CQL_PORT, CQL_PORT);
}
```

Comment (farodin91) [reply janusgraph-cql/src/test/java/org/janusgraph/JanusGraphCassandraContainer.java]: Fixed

**Comment by farodin91** [comment]
This code has only impact on the hadoop part which is not tested using full build.

**Comment by farodin91** [comment]
Build passed: https://travis-ci.com/farodin91/janusgraph/builds/123426899142d9235d972

**Comment by porunov** [comment]
@farodin91 You can add `CTR` to commit message and merge it with `CTR`

**Comment by farodin91** [comment]
Closed by me. I maded a CTR commit on master and 0.4

**Comment by porunov** [comment]
This PR is reverted

**Pull request closed without merging**